src/getCoinCombination.test.js

```diff
@@ -3,8 +3,33 @@
 describe('getCoinCombination', () => {
   const { getCoinCombination } = require('./getCoinCombination');
 
-  it(`should be declared`, () => {
+  it(`function should be declared`, () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
```

Comment:
It would be nice to add checks of each of the coins returned when a certain amount of cents is put - you have the test for pennie, but also add a check for when the input is 5 and 10 cents, and also check some big input - which will use all of the coins - for example '69'.